src/Sylius/Bundle/ApiBundle/Controller/Payment/GetPaymentConfiguration.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Controller\Payment;
+
+use Sylius\Bundle\ApiBundle\Provider\PaymentConfigurationProvider;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/** @experimental */
+final class GetPaymentConfiguration
+{
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+
+    /** @var PaymentConfigurationProvider */
+    private $paymentConfigurationProvider;
+
+    public function __construct(
+        OrderRepositoryInterface $orderRepository,
+        PaymentConfigurationProvider $paymentConfigurationProvider
```

Comment:
Payment configuration is an implementation of the class, not a contract. We should introduce an interface for it.

Reply:
Maybe we can use the interface, that you have introduced?